source/fuzz/fuzzer_util.cpp

```diff
@@ -418,6 +418,8 @@ bool IsValidAndWellFormed(const opt::IRContext* ir_context,
                           spv_validator_options validator_options,
                           MessageConsumer consumer) {
   if (!IsValid(ir_context, validator_options, consumer)) {
+    // Expression to dump |ir_context| to /data/temp/shader.spv:
+    //    DumpShader(ir_context, "/data/temp/shader.spv")
```

Comment:
Debug code to remove?

Reply:
I actually decided to include this as it's very useful (especially as someone new to the project wouldn't know about DumpShader).